.templates/openstack/disk-pools.yml

```diff
@@ -0,0 +1,23 @@
+---
+disk_pools:
+  - name: tiny
```

Comment:
Do you mind removing the `tiny` disk pool? It's not used anywhere on these templates, and we will maintain consistency with other templates.